xmake/toolchains/llvm/xmake.lua

```diff
@@ -31,8 +31,8 @@ toolchain("llvm")
     -- set toolset
     set_toolset("cc",     "clang")
     set_toolset("cxx",    "clang", "clang++")
-    set_toolset("mxx",    "clang", "clang++")
-    set_toolset("mm",    "clang", "clang++")
+    set_toolset("mxx",    "clang++", "clang")
+    set_toolset("mm",     "clang")
```

Comment:
and here

Reply:
if i remove it, i'm getting "error: cannot get program for mxx"